### DISCUSSION TITLE
Use execute(-silent) on all selected lines

### DISCUSSION THREAD

**Discussion by fritzrehde** [junegunn/fzf#2773] [Q&A] [answered]
Is it possible to use the execute command (which can be mapped to a keybinding) to all currently selected lines?

**Comment by junegunn** [comment, marked as answer]
Yes, with `{+}`.

```sh
fzf -m --bind 'space:execute:echo {+}; read'
```

See "PREVIEW" section of the man page.

> A  placeholder expression starting with + flag will be replaced to the space-separated list of the selected lines (or the current line if no selection was made) individually quoted.
